web-client/src/views/CaseDetail/DeficiencyStatisticsForm.tsx

```diff
@@ -91,30 +96,27 @@ export const DeficiencyStatisticsForm = connect(
                 </FormGroup>
               )}
 
-              {form.yearOrPeriod === 'Period' && (
-                <FormGroup errorText={validationErrors.lastDateOfPeriod}>
-                  <DateInput
-                    id="last-date-of-period"
-                    label="Last date of period"
-                    names={{
-                      day: 'lastDateOfPeriodDay',
+              {/*
+                                day: 'lastDateOfPeriodDay',
```

Comment:
is this left included for any specific reason?

Reply:
Nope! Just forgot to delete it, thanks.